app/api/health_api.py

```
@@ -4,9 +4,9 @@
 router = APIRouter()
 
 @router.get("/health")
-def health_check():
+async def health_check():
     try:
-        db.command("ping")
+        await db.command("ping")
         return {"status": "MongoDB connected"}
-    except:
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)